<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Model">
      <attribute name="title">
       <string>Model</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="enableButton">
           <property name="text">
            <string>enable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disableButton">
           <property name="text">
            <string>disable</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Add">
      <attribute name="title">
       <string>Add</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>95</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Config">
      <attribute name="title">
       <string>Config</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
